### PR TITLE
DOC: minor doc improvements to advanced solvers

### PR DESCRIPTION
**Pull request by adler-j**

### CODE REVIEW

File: odl/solvers/advanced/chambolle_pock.py [hunk 1]
```diff
@@ -127,7 +127,10 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
 
     See Also
     --------
-    douglas_rachford_pd : Solver for similar problems.
+    douglas_rachford_pd : Solver for similar problems but can handle infimal
+                          convolutions and several forward operators.
```
Comment (kohr-h) [inline odl/solvers/advanced/chambolle_pock.py]: several -> multiple

File: odl/solvers/advanced/chambolle_pock.py [hunk 2]
```diff
@@ -127,7 +127,10 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
 
     See Also
     --------
-    douglas_rachford_pd : Solver for similar problems.
+    douglas_rachford_pd : Solver for similar problems but can handle infimal
```
Comment (kohr-h) [inline odl/solvers/advanced/chambolle_pock.py]: that can additionally handle

File: odl/solvers/advanced/chambolle_pock.py [hunk 3]
```diff
@@ -127,7 +127,10 @@ def chambolle_pock_solver(op, x, tau, sigma, proximal_primal, proximal_dual,
...
+                          convolutions and several forward operators.
+    forward_backward_pd : Solver for similar problems but handles a
+                          differentiable term.
```
Comment (kohr-h) [inline odl/solvers/advanced/chambolle_pock.py]: same comment as above

File: odl/solvers/advanced/douglas_rachford.py [hunk 4]
```diff
@@ -114,7 +115,9 @@ def douglas_rachford_pd(x, prox_f, prox_cc_g, L, tau, sigma, niter,
 
     See Also
     --------
-    chambolle_pock : Solver for similar problems.
+    chambolle_pock_solver : Solver for similar problems.
+    forward_backward_pd : Solver for similar problems but handles a
+                          differentiable term.
```
Comment (kohr-h) [inline odl/solvers/advanced/douglas_rachford.py]: see above

File: odl/solvers/advanced/forward_backward.py [hunk 5]
```diff
@@ -128,6 +129,14 @@ def forward_backward_pd(x, prox_f, prox_cc_g, L, grad_h, tau, sigma, niter,
     where, if the simpler problem is considered, all :math:`\\nu_i` can be
     considered to be :math:`\\infty`.
 
+    See Also
+    --------
+    chambolle_pock_solver : Solver for similar problems.
+    douglas_rachford_pd : Solver for similar problems but can handle infimal
+                          convolutions and several forward operators.
```
Comment (kohr-h) [inline odl/solvers/advanced/forward_backward.py]: see above

**Comment by kohr-h** [comment]
Small things, good otherwise.

**Comment by adler-j** [comment]
Merge after CI.

**Pull request merged**